generators/app/index.js

```diff
@@ -404,6 +413,55 @@ module.exports = generators.Base.extend({
           }
         }
       }.bind(self));
+    },
+
+    addGitignore: function (self, cb) {
+      self = self || this;
+      cb = cb || this.async();
+      var cssCompiledPath = self.staticsRoot + '/' + (self.cssCompiledOutput).split('/')[0],
+          jsCompiledPath = self.staticsRoot + '/' + (self.jsCompiledOutput).split('/')[0],
+          whenIgnoring = function () {
+            return self.gitIgnore;
+          };
+
+      return self.prompt(
+        [
+          {
+            when: whenIgnoring,
+            type: 'input',
+            name: 'gitIgnoreStream',
+            message: 'What location would you like you add to .gitignore?',
```

Comment:
Adding to the .gitignore at this point in app creation is pretty niche, moreso than setting the public and .build directories, and those don't have prompts.

I think it would be better to add gitignore as an option rather than a recursive prompt to keep things simple for this reason.